### DISCUSSION TITLE
「提问」如何看待《黑神话：悟空》成为steam单机游戏在线人数第一

### DISCUSSION THREAD

**Discussion by cyq20091029** [share121/inter-knot#3286] [其他]
<img width="218" alt="截屏2024-08-19 17 19 34" src="https://github.com/user-attachments/assets/f9019b56-e5d0-48d4-82f1-fa533e1fc64c">

欢迎各位绳匠讨论

**Comment by tayiie** [comment]
# 今天不上班的富哥真多，慕了

**Reply by RaphaelHider** [reply]
> # 今天不上班的富哥真多，慕了

我同事请两天假玩的，我只能说6

**Comment by RaphaelHider** [comment]
坐着看

**Reply by cyq20091029** [reply]
真就看待类帖子万能答案呗

**Comment by yzysrewrwqr** [comment]
<img src="https://img-static.mihoyo.com/communityweb/upload/a7304904a31329464114b1074218967e.png" alt="帕姆 瘫倒" width="75" >早知道我也请假

**Comment by sane520** [comment]
躺着看
